packages/notifi-react-card/lib/components/defaults.css

```diff
@@ -482,7 +483,7 @@ input:-webkit-autofill {
   font-size: 16px;
   line-height: 20px;
   cursor: pointer;
-  color: var(--notifi-font-color);
+  color: #ffffff;
```

Comment:
Should we have a --notifi-button-font-color that defaults to white then?

Reply:
sure